scss/utilities/kss/_kss.scss

```diff
@@ -0,0 +1,13 @@
+@charset "UTF-8";
+
+//
+// KSS Fullscreen Scrolling Fix
+//
+// Adding 2px to the default kss fullscreen padding. This resolves the 1-2px horizontal scrolling issue.
+//
+// Style guide: KSS.ScrollFix
+//
+
+#kss-node .kss-section.is-fullscreen {
```

Comment:
This is specific to the styleguide and does not belong here. Please remove.

Reply:
I'll work on that shortly.